switchcraft/data/computercraft/lua/rom/modules/main/chatbox.lua

```diff
@@ -1,10 +1,12 @@
 -- SwitchCraft chatbox legacy handler by Lemmmy
 -- Contact Lemmmy#4600 regarding any issues
 
-local chatbox = {}
+local chatbox = {} 
 
+---Websocket URI for the chatbox serveratbox = {}
 chatbox.SERVER_URL = "wss://chat.sc3.io"
 
+---Error messages for the chatbox server
```

Comment:
Shouldn't this be `URL` instead of `URI`? IIRC they're 2 completely separate things